app/views/foreman_statistics/trends/index.html.erb

```diff
@@ -2,15 +2,6 @@
 <% title _("Trends") %>
 <% title_actions new_link(_("Add Trend Counter")),
                  documentation_button('4.1.3Trends') %>
-<% if @trends.empty? %>
-  <%= alert :class => 'alert-info', :header => _("No trend counter defined"),
-            :text => (_("To define trend counters, use the Add Trend Counter button.</br> To start collecting trend data, set a cron job to execute 'foreman-rake foreman_statistics:trends:counter' at least every %s minutes.") % Setting[:outofsync_interval]).html_safe %>
-<% end %>
```

Comment:
In general it's good to keep an alert that it's unconfigured, just drop the cronjob part of the sentence. Other pages do this too. You could perhaps hide the table if there are no trends? Or put it in the tbody.

Reply:
Yeah, I'd agree, but it seems we have a welcome page for that:
https://github.com/theforeman/foreman_statistics/blob/0b3685c15f8da14f55757ad6eed332a9b74097b7/app/views/foreman_statistics/trends/welcome.html.erb
I'd say this alert is dated before the welcome pages and it was just forgotten there.